scipy/stats/__init__.py

```diff
@@ -201,6 +201,7 @@
    mvsdist
    entropy
    median_absolute_deviation
```

Comment:
you can remove this, best to only list the non-deprecated function

Reply:
you may then need to add it to the skiplist in `refguide_check.py` (not sure, could be it skips deprecated functions already)

Reply:
Removed. 

> you may then need to add it to the skiplist in `refguide_check.py` (not sure, could be it skips deprecated functions already)

It looks like `refguide_check.py` skips them already: 
this function splits them out (deprecated are determined by executing with empty dict and checking if a deprecated warning is thrown), 
https://github.com/scipy/scipy/blob/c1372d8aa90a73d8a52f135529293ff4edb98fc8/tools/refguide_check.py#L877 
then two lines here:
https://github.com/scipy/scipy/blob/c1372d8aa90a73d8a52f135529293ff4edb98fc8/tools/refguide_check.py#L881-L882
the first does a count of deprecated and not and the second does a set difference, removing the deprecated, so we should be good here, unless I missed something :).